recipes/libcomcat/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set version = "1.1" %}
+
+package:
+  name: libcomcat
+  version: {{ version }}
+
+source:
+  url: https://github.com/usgs/libcomcat/archive/{{ version }}.tar.gz
+  sha256: f53c191a1ea2d389d973b45d80295e4b6fca90a779a6d15b4b210380e385f384
+
+build:
+  number: 0
+  skip: True  # [py27]
+  script: python -m pip install --no-deps --ignore-installed .
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
```

Comment:
Is `obspy` indirectly included here through another requirement?

https://github.com/usgs/libcomcat/blob/master/libcomcat/utils.py#L12

Because importing the module no-ops, it's hard to tell if you have all of the requirements.

Reply:
@synapticarbors That's an oversight.  Sigh.  Incidentally, what is the best source for documentation on these meta.yaml files?  There are a bunch of things I've set in here that I don't really understand...

Reply:
@synapticarbors Also, I don't understand what this part of your question means: "Because importing the module no-ops..."

Reply:
I tend to look at:

https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html

Were there specific things you don't understand? If you don't see the answer in the docs, don't hesitate to ask and I can try to help.

Reply:
When you import `libcomcat`, its `__init__.py` file doesn't import anything else, so you miss if a dependency is missing. When I said "no-ops", I just meant that it doesn't execute any additional code. Some python packages import submodules in the package's primary `__init__`, so you end up initiating a bunch of imports of dependencies throughout the package. Either way of setting things up is fine, you just have to be careful since the simple import test in the `meta.yaml` doesn't tell you much out than the package exists.

Reply:
Ok, so here:
https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#skipping-builds

Does "skip: True  # [not win]"  mean "skip building for platforms that are not windows"?  

Is there a list of valid things that can be inside the brackets here?

Reply:
Correct. The list is here:

 https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#preprocessing-selectors